### PR TITLE
New smc version

### LINKED ISSUE #1580: Update smc to v0.6.0.0

### PR DESCRIPTION
**Pull request by ARR552**
Closes #1580

### What does this PR do?

Updates smc to v0.6.0.0

### Reviewers

- @cool-develope

### CODE REVIEW

**Comment by cool-develope** [comment]
AFAIK, these name changes could impact the genesis contract name

File: config/environments/public/public.node.config.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ EnableLog = false
 MaxConns = 200
 
 [Pool]
-FreeClaimGasLimit = 150000
+FreeClaimGasLimit = 1500000
```
Comment (cool-develope) [inline config/environments/public/public.node.config.toml:16]: any reason for the increment?

**Review by cool-develope — APPROVED** [review]
LGTM!

File: config/environments/public/public.node.config.toml (hunk 1 above)
Comment (ARR552) [reply config/environments/public/public.node.config.toml:16]: Yes, due to some smc updates, right now the claim can be use more gas during an erc20 bridge where the erc20 doesn't exist yet on l2 and the contract needs to be deployed

**Pull request merged**